README.md

```diff
@@ -59,9 +61,47 @@ pip install -e .
 
 ## Usage
 
-### Convert to required format
+### Conversion to Markdown format
 
-TODO
+Two kind of sources are minimal required, one are the requirements (*.trlc) and the model (*.tls). Both can be added by file name or just the path where they are located.
+
+```bash
+pyTRLCConverter -s doc/models -s doc/sw-requirement markdown
+```
+
+It will create a Markdown file with the same name than the requirements file (*.trlc) in the current directory, but with the Markdown extension (.md).
+
+If the requirements are splitted in several files, a Markdown file will be created for each.
+
+No additional Markdown converter specific arguments are supported yet.
+
+More examples are shown in the [examples folder](./examples/).
+
+### Conversion to docx format
+
+Similar to the Markdown conversion, minimal required are the requirements (*.trlc) and the model (*.tls). Both can be added by file name or just the path where they are located.
+
+```bash
+pyTRLCConverter -s doc/models -s doc/sw-requirement docx
+```
+
+It will create a docx file with default name ```output.docx``` in the current directory.
+
+If the requirements are splitted in several files, they will be all part of a single docx file.
+
+Have a look to the additional arguments:
```

Comment:
Better: 

The docx converter supports additional arguments that are shown by  adding the ```--help``` option **after** the ```docx``` subcommand.

```bash
pyTRLCConverter docx --help
usage: pyTRLCConverter docx [-h] [-t TEMPLATE] [-n NAME]

options:
  -h, --help            show this help message and exit
  -t TEMPLATE, --template TEMPLATE
                        Load the given docx file as a template to append to.
  -n NAME, --name NAME  Name of the generated output file inside the output folder (default = output.docx).
```